### PR TITLE
Added AU_MEDICARE test case

### LINKED ISSUE #4: Detecting AU_MEDICARE through Patterns, Context, and Checksum
An Australian Medicare number is a type of Personal Identification Information (PII) used in the Australian Medicare system. With this number, you can identify if someone has opted into the system, gain access to other insurance plans, and ensure Medicare pays first on the bill. Without this number, one would lose access to their insurance and have to pay medical bills entirely with their own money. If someone steals the number, they can rack up charges with insurance providers under someone else's name. If someone's medicare number is breached, it puts their insurance and savings at risk, making it very important to keep protected.

### PR DESCRIPTION
**Pull request by emsmdm**

### CODE REVIEW

**Review by claesmk — APPROVED** [review]
Looks good

**Comment by claesmk** [comment]
@emsmdm this is waiting on a review from your team.  After that you can merge it

**Review by ZionPacker — APPROVED** [review]
Looks well written and able to be merged

**Pull request merged**
